### DISCUSSION TITLE
Welcome to Peakaboo Discussions!

### DISCUSSION THREAD

**Discussion by nsherry4** [nsherry4/Peakaboo#278] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by oliverP222** [comment]
Hello, my name is Oliver and I am using a Bruker Tracer 5i XRF for analyzing artifacts.  I would love to try out Peakboo to see if I can look at the spectra produced.  I see it does not support the Bruker .pdz file format and so used read-pdz, a software program that converts the .pdz file into a .csv file of the spectrum.  I hoped Peakaboo could open this but it fails too.  Any suggestions would be greatly appreciated.
